src/mapnik_vector_tile.cpp

```diff
@@ -3348,7 +3271,7 @@ void VectorTile::EIO_IsSolid(uv_work_t* req)
 {
     is_solid_vector_tile_baton_t *closure = static_cast<is_solid_vector_tile_baton_t *>(req->data);
     try {
-        closure->result = mapnik::vector_tile_impl::is_solid_extent(closure->d->get_tile(),closure->key);
+        closure->result = mapnik::vector_tile_impl::is_solid_extent(detail::get_tile(closure->d->buffer_),closure->key);
```

Comment:
No test case still for exception case here, hard to write test perhaps as buffer would have to contain an invalid command.